lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/report/edit/ReportEditScreen.kt

```diff
@@ -0,0 +1,355 @@
+    ) {
+        item {
+            Column(
+                verticalArrangement = Arrangement.spacedBy(8.dp)
+            ) {
+                Text(stringResource(MR.strings.title))
+                OutlinedTextField(
+                    modifier = Modifier.fillMaxWidth(),
+                    value = uiState.reportOptions2.title,
+                    label = null,
+                    singleLine = true,
+                    onValueChange = { newTitle ->
+                        val updatedOptions = uiState.reportOptions2.copy(title = newTitle)
+                        onReportChanged(updatedOptions)
+                    },
+                    keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Text),
+                    supportingText = {
+                    }
+                )
+            }
+        }
+        item {
+            LabeledDropdownMenu(
+                selectedValue = uiState.reportOptions2.xAxis ?: 0,
+                label = stringResource(MR.strings.x_axis),
+                options = ReportXAxisConstants.X_AXIS_OPTIONS,
+                onOptionSelected = {
+                    val updatedOptions = uiState.reportOptions2.copy(xAxis = it.value)
+                    onReportChanged(updatedOptions)
+                },
+            )
+        }
+
+        // Dynamically iterate over the series
+        uiState.reportOptions2.series.forEach { seriesItem ->
+            item {
+                HorizontalDivider(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .defaultScreenPadding(), thickness = 1.dp
+                )
+            }
+            item {
+                Column(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    verticalArrangement = Arrangement.spacedBy(8.dp)
+                ) {
+                    // Series Title Input
+                    Text(
+                        stringResource(MR.strings.series_title),
+                    )
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.SpaceBetween,
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        OutlinedTextField(
+                            modifier = Modifier.weight(1f),
+                            value = seriesItem.reportSeriesTitle,
+                            label = null,
+                            singleLine = true,
+                            onValueChange = { newTitle ->
+                                val updatedSeries = seriesItem.copy(reportSeriesTitle = newTitle)
+                                onSeriesChanged(updatedSeries)
+                            },
+                        )
+                        Icon(
+                            imageVector = Icons.Filled.Close,
+                            contentDescription = "Remove filter",
```

Comment:
contentDescriptions are used by text-to-speech tools - so they must use localized strings (eg stringResource) the same as text that appears 'normally'.